qiskit_experiments/data_processing/nodes.py

```diff
@@ -34,65 +37,67 @@ def __init__(self, axis: int, validate: bool = True):
         super().__init__(validate)
         self._axis = axis
 
-    def _format_data(self, datum: Any, error: Optional[Any] = None):
-        """Format the data into numpy arrays."""
-        datum = np.asarray(datum, dtype=float)
+    def _format_data(self, data: np.ndarray) -> np.ndarray:
+        """Format the data into numpy arrays.
+
+        Args:
+            data: A full data array to format.
```

Comment:
The word `full` needs some clarification.

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/481/commits/811e87f0c5800b79e2627ebb946cec49f77d30c8

Reply:
This looks good:
```
            data: A data array to format. This is a single numpy array containing
                all circuit results input to the data processor.
```
Why not use the same language in the `nodes.py`? I find that `all-result` also needs a word of clarification.

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/481/commits/1b3aea08e18fe0b2e017aee5a7cce00e891f8c83